ha/monitor/k8s/parser.py

```diff
@@ -141,11 +146,13 @@ def _create_health_alert(self, res_type, res_name, health_status, generation_id)
         health_status: health of that resource. Ex: online, failed
         generation_id: name of the node in case of node alert
         """
-        self.event = HealthEvent()
-        self.event.set(HealthAttr.RESOURCE_TYPE.value, res_type)
-        self.event.set(HealthAttr.RESOURCE_ID.value, res_name)
-        self.event.set(HealthAttr.NODE_ID.value, res_name)
-        self.event.set(HealthAttr.RESOURCE_STATUS.value, health_status)
+        self.payload[HealthAttr.RESOURCE_TYPE.value] = res_type
+        self.payload[HealthAttr.RESOURCE_ID.value] = res_name
+        self.payload[HealthAttr.NODE_ID.value] = res_name
+        self.payload[HealthAttr.RESOURCE_STATUS.value] = health_status
+        # Create event schema object with payload data also
+        # can change value of payload attributes with set function
+        self.event = HealthEvent(**self.payload)
```

Comment:
why not called KvPayload(self.payload) or is it getting called internally.
I have approved it is just a query.

Reply:
yes its handle internally in HealthEvent 

`def __init__(self, **kwargs):
        """Initialize payload.
           Param: kwargs = key, value pairs"""
        payload = KvPayload()
        for key in HealthEvent._health_attrs:
            payload.set(str(key), '')
        for key, val in kwargs.items():
            payload.set(key, val)
        payload.set('specific_info', '')
`